apps/ca.c

```diff
@@ -200,7 +200,7 @@ const OPTIONS ca_options[] = {
     {"rand_serial", OPT_RAND_SERIAL, '-',
      "Always create a random serial; do not store it"},
     {"multivalue-rdn", OPT_MULTIVALUE_RDN, '-',
-     "Enable support for multivalued RDNs"},
+     "Enable support for multi-valued RDNs. This option is meanwhile always on"},
```

Comment:
This "meanwhile" might be an artefact from German "mittlerweile", but in any case, in English it doesn't make sense. Is it supposed to say "By default, this option is always on"?

Reply:
@h-vetinari, thanks for your comments.
"By default" does not really hit the point because it this option cannot be disabled any more.
I doubt if it is worth doing such a correction here now since this is already approved - @t8m, what do you think?

If so, @h-vetinar please provide a better wording, or how about replacing the whole string by:
"This has become obsolete since support for multi-valued RDNs is now always on."

We might also simply drop this option entirely from the help output (also for the `req` app).

Reply:
+1 to drop the option from the help output. It is already long enough and the deprecation is mentioned in the manual page.

Reply:
I was about to push that, 
but then I realized that this would mean that old CLI invocations using `-multivalue-rdn` would start failing. 
So I just changed the help text using my own proposal. 
Is the English fine now? Then this should be ready for re-approval.